tests/lib/__init__.py

```diff
@@ -907,3 +907,11 @@ def need_mercurial(fn):
     return pytest.mark.mercurial(need_executable(
         'Mercurial', ('hg', 'version')
     )(fn))
+
+
+def is_svn_installed():
```

Comment:
Can you put this after `is_bzr_installed()` since the functions have the same structure?